.spelling

```diff
@@ -585,3 +585,7 @@ AddPiece
 PreCommit1
 U.2
 Hoejsbo
+ - content/en/storage-provider/infrastructure/storage/index.md
+VDEVs
+resilver
+Resilvering
```

Comment:
@johnnymatthews either I am learning two new words here or these are typos. For context, they were used in this paragraph https://github.com/filecoin-project/filecoin-docs/pull/1978/commits/f3b4872b7ec407e1d940998818b8e7a463db855f#diff-b9f713afb0831a303a717e2e45c55788e5f192f10d82d809b822097c559524e3L28-R28

Merriam-Webster says https://www.merriam-webster.com/dictionary/resilver, which makes me think this is a typo. But maybe this word means different things elsewhere / in other contexts?

Here's one guess:

resilver -> resolver? But in that case, resilvering -> resolvering, which doesn't seem right

Reply:
Actually, I guess this could make sense given the Merriam definitions and the sentence context: 

Since resilver more or less means "rebuild/recover/renew a silver thing", that's sort of what the sentence is saying "Resilvering (or rebuilding) is the I/O intensive process that takes place when a disk in a RAID group is replaced (typically after a disk failed)"

So I guess it makes sense, sort of. However, if it's not some sort of SP-specific technical term, should probably just be "rebuild" and "rebuilding". If it is a technical term, should probably be a tooltip link or at least have a Glossary entry or link out to the term definition somewhere. I figure I won't be the only reader confused by this

Reply:
Yeah according to [lonesysadmin.net](https://lonesysadmin.net/2012/03/23/why-is-it-called-resilvering/):

> Q: Why do some people refer to the process of remirroring or rebuilding a RAID 1 drive set as “resilvering?”
> 
> A: Antique mirrors (the reflective kind you hang on a wall, or are in your bathroom) used silver (Ag) for the reflective coating, below the glass. Over time that silver would get tarnished and/or damaged, so you’d restore them by re-silvering them. I’m sure you’ve all seen this, where an old mirror has streaks in it but they’re below the glass.
> 
> When your RAID 1 mirror set gets “tarnished” you resilver it and it’s shiny & new again. You can rebuild a RAID 5 array but you resilver a mirror. :)

But still, resilvering is a silly word. Let's just rewrite the sentence to use _rebuild_.